docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -1195,358 +1195,791 @@ thor:
+          Here we have hpcc2.roxie making a call to hpcc1.roxie in another 
+          environment, and we want them to trust each other. There are two main configuration
+          customizations needed to set up this zone
+          of trust. Each of the two HPCC environments have these changes.</para> 
 
-      <sect3 id="Container_Certificates">
-        <title>Enable Certificates</title>
+          <para>First, the remote issuer is enabled and its spec set to use the Vault.
+          Both HPCC environments share the same Vault server, which allows them to use
+          the same underlying certificates to establish trush. Note alseo that the
+          <emphasis>ca</emphasis> property must be set to null to override the default.</para>
 
-        <para>Use the certificates section to enable cert-manager to generate
-        TLS certificates for each component in the HPCC Systems
-        deployment.</para>
+          <programlisting>
+            spec:
+              ca: null
+              vault:
+                path: pki/sign/hpccremote
+                server: http://vault.vaultns:8200
+                auth:
+                  tokenSecretRef:
+                      name: cert-manager-vault-token
+                      key: token
+          </programlisting>
 
-        <programlisting>certificates:
-  enabled: false
-  issuers:
-    local:
-      name: hpcc-local-issuer</programlisting>
+          <para>Second, each roxie must be told to trust the other. In the hpcc2 values file the
+          roxie service has its trustClients array set to include hpcc1.roxie, and likewise the
+          hpcc1 values file has its roxie service trustClients array set to include hpcc2.roxie.
+          These files can be found in the repo at helm/examples/vault-pki-remote/values-hpcc2.yaml
+          and helm/examples/vault-pki-remote/values-hpcc1.yaml.
+          Here are the relevant parts of the hpcc2 values file:
+          </para>
 
-        <para>In the delivered YAML file certificates are not enabled, as
-        illustrated above. You must first install the cert-manager to use this
-        feature.</para>
-      </sect3>
+          <programlisting>
+            roxie:
+            - name: roxie2
+              disabled: false
+              services:
+              - name: roxie2
+                servicePort: 29876
+                visibility: local
+                trustClients:
+                - commonName: roxie1.hpcc1
+          </programlisting>
 
-      <sect3 id="Certificate_Issuers_Configuration">
-        <title>Certificate Issuers Configuration</title>
+          <sect5 id="InstallHashicorpVault">
+            <title>Install Hashicorp vault service in dev mode</title>
 
-        <para>Certificate issuers are configured in the
-        <literal>certificates.issuers</literal> section of the Helm values.
-        The system supports multiple issuer types:</para>
+            <para>This is for development only, never deploy this way in
```

Comment:
suggestion: (for clarity) This scenario is for development only, you would never want to deploy this way in a real world production environment.